config/default/kustomization.yaml

```diff
@@ -7,6 +7,7 @@ commonLabels:
 resources:
 - namespace.yaml
 - credentials.yaml
+- metrics_service.yaml
```

Comment:
Why is this in default, you might ask? Because it appears that we only run config/default kustomize. That uses `bases` so it should have loaded the kustomize file from under the referenced folders, but it doesn't for some reason which is a different matter all together.

For now, since this should be the default behaviour anyways, I think it's fine to add it here.